### DISCUSSION TITLE
Dúvida

### DISCUSSION THREAD

**Discussion by EdeysonGomes** [mate28-ic-ufba/turma-20212#54] [Q&A]
Pessoal, como eu vejo o **crescimento** de um projeto Git em função do tempo?

Grato.

**Comment by lidianycs** [comment]
Coloquei a primeira e última versão no https://codetabs.com/count-loc/count-loc-online.html e comparei as informações.

**Reply by christinaflachufba** [reply]
Oi Edeyson,

Consultou os INSIGHTS do projeto?

**Reply by christinaflachufba** [reply]
No Ohloh, tem uma parte de Comparação entre projetos e Code Analysis -- ver Lines of Code / View as graph. Talvez ajude.
